test/MochaSpecRunner.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mocha Tests</title>
+    <script type="text/javascript" src="lib/webcomponents.js"></script>
+    <link href="lib/mocha.css" rel="stylesheet" />
+    <link rel="import" href="../lab/polymer/components/my-menu-button.html">
+</head>
+
+<body>
+    <my-menu-button style="position: absolute; top: 0; right: 0" href="subpage.html">Distributed <strong>Content</strong></my-menu-button>
```

Comment:
I think we cannot merge until this is clarified. @alshakero can you pls check what exactly is the reason for these warnings?